content/pt-br/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,182 @@
+---
+title: EndpointSlices
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="stable" >}}
+
+_EndpointSlices_ entregam uma maneira simples de rastrear endpoints de rede através de um cluster Kubernetes. Eles oferecem uma alternativa mais escalável e extensa aos Endpoints.
```

Comment:
How about using "fornecem" instead of "entregam"?

Reply:
I also think in the context of this translation "dentro de um cluster Kubernetes" is better than "através de um cluster Kubernetes".